src/components/mock/payment.js

```diff
@@ -0,0 +1,8 @@
+import {v4 as uuidv4} from 'uuid' 
+
+export const mockTeachers=[
+    {_id:uuidv4(),payMethod:"credit-card",studentId:"nihnoi",tutorshipId:"ythr",comment:"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo reprehenderit fugiat aut quis,"},
```

Comment:
when you have multiple keys in one objects is proper to order in different lines to easy read

For Example:

`{
      id: uuid(),
      type: 'freelancer_basic',
      name: 'Freelancer Basic Subscription',
      image: '/images/products/product_freelancer.svg',
      subscriptions: '13,153',
      currency: '$',
      price: '5.00',
      progress: 93
    },`

Reply:
Donde